src/results/ResultsContainer.tsx

```diff
@@ -120,38 +121,48 @@ export class Results extends Component<IResultsProps, ResultsContainerState> {
       dispatchAddFacet,
       dispatchRemoveFacet,
       dispatchUpdateColumnSort,
+      namespace,
       sort,
     } = this.props;
     const { selectedRows } = this.state;
     if (isFetching) {
       return <h3>Loading...</h3>;
     }
+    const info = infoMappings[namespace];
```

Comment:
Here maybe `info` can be further destructed to `{ name, links }`. If we stick to this pattern it will help with more complicated object, etc.

Reply:
true, done